pydantic_factories/constraints/decimal.py

```diff
@@ -76,8 +76,6 @@ def handle_constrained_decimal(field: "ConstrainedDecimal") -> Decimal:
     multiple_of = cast("Optional[Decimal]", field.multiple_of)
     decimal_places = field.decimal_places
     max_digits = field.max_digits
-    if multiple_of == 0:
```

Comment:
multiple_of == 0 should generate ParameterError. See #120

This is the reason why I removed this bit + change tests for all numbers